p01-data-binding/src/Content/Content.js

```diff
@@ -1,37 +1,65 @@
+    showEditor: true,
+  });
 
-        Why do we use it?
-        It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
+  function handleToggleClick() {
+    setState({ ...state, showEditor: !state["showEditor"] });
+  }
 
+  function handleChange(e) {
+    const { name, value } = e.target;
+    setState({
+      ...state,
+      [name]: value,
+    });
+  }
 
-        Where does it come from?
-        Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
+  return (
+    <div className="container">
+      {state["showEditor"] ? (
+        <div className="card editor">
+          <div className="form-group">
+            <label>Title</label>
+            <input
+              type="text"
+              name="title"
+              value={state["title"]}
+              onChange={handleChange}
+            />
+          </div>
+          <div className="form-group">
+            <label>Description</label>
+            <textarea
+              rows={7}
+              name="description"
+              value={state["description"]}
+              onChange={handleChange}
+            />
+          </div>
+        </div>
+      ) : null}
 
-        The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.
-        </p>
+      <div className="card content">
+        <h3>{state["title"]}</h3>
+        <p>{state["description"]}</p>
+        <button onClick={handleToggleClick} name="showEditor">
+          {state["showEditor"] ? "Hide Editor" : "Show Editor"}
```

Comment:
Good use of ternary operators, another way you can write it could be:

```
{ state.showEditor ? "Hide" : "Show"} {" Editor" }
or using string templates
{ `${state.showEditor? "Hide" : "Show"} Editor` }
```

There is no wrong way to do it, just different ways of do it